src/polymatheia/data/reader.py

```diff
@@ -322,3 +355,56 @@ def __iter__(self):
         if self._file.seekable():
             self._file.seek(0)
         return NavigableDictIterator(iter(DictReader(self._file)))
+
+
+class SRUExplainRecordReader(object):
+    """The class:`~polymatheia.data.reader.SRUExplainRecordReader` is a container for SRU Explain Records."""
+
+    def __init__(self, url):
+        """Construct a new class:`~polymatheia.data.reader.SRUExplainRecordReader`.
+
+        :param url: The base URL of the SRU server
+        :type url: ``str``
+        """
+        self._url = url
+        self._explain = SRUpy(self._url).explain()
+        self.schemas = [(schema["@name"], schema.title)
+                        for schema in NavigableDict(self._explain).explain.schemaInfo.schema]
+        self.echo = NavigableDict(self._explain.echo)
+
+    def __iter__(self):
+        """Return this :class:`~polymatheia.data.reader.SRUExplainRecordReader`` as the iterator."""
+        return NavigableDictIterator(iter(self._explain),
+                                     mapper=lambda record: {record[0]: record[1]}
+                                     )
+
+
+class SRURecordReader(object):
+    """The :class:`~polymatheia.data.reader.SRURecordReader` is an iteration container for Records fetched via SRU.
+
+    The underlying library (SRUpy) automatically handles the continuation parameters, allowing for simple iteration.
+    """
+
+    def __init__(self, url, **kwargs):
```

Comment:
While passing kwargs is a sensible way of accessing extended parameters, the core parameters should be listed and documented as explicit parameters. The main reason for this is to make it easier for novice users to understand what parameters can be used and also for IDEs to provide support on what parameters are available.

The parameters should also have the same names as in existing readers (for example max_records).